### PR TITLE
Adds How-To for using OpenMP to calculate PI

### PR DESCRIPTION
**Pull request by gnikit**
This PR merges into the refactoring branch #127 (part of the changes for the OpenMP blog).
I added a modified version of the https://github.com/gjbex/Fortran-MOOC/blob/master/source_code/computing_pi/compute_pi_omp.f90 code to demo the metapackages in a more realistic usecase.
Let me know what you think.

### CODE REVIEW

**Review by perazz — APPROVED** [review]
Nice work @gnikit, looks good to me. Thank you!

File: pages/how-to/compute-pi-openmp.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+This is a simple example of how to use OpenMP with fpm.
+It is an adapted version of the OpenMP example that can be found
+[here](https://github.com/gjbex/Fortran-MOOC/blob/master/source_code/computing_pi/compute_pi_omp.f90) under a
+[CC-BY-4.0](https://github.com/gjbex/Fortran-MOOC/blob/master/LICENSE) license.
+
+The code approximates the value of PI by performing parallelized numerical
+integration over a quarter of the unit circle.
+The code is structured as follows:
+
+```{literalinclude} ../../src/how-to/copmute-pi-openmp/app/main.f90
```
[the remark this answers is not in the document]
Comment (gnikit) [reply pages/how-to/compute-pi-openmp.md:12]: Awesome thanks will fix now. I am curious how my local rendering of the docs worked

Comment (gnikit) [inline pages/how-to/compute-pi-openmp.md:12]:
```suggestion
```{literalinclude} ../../src/how-to/compute-pi-openmp/app/main.f90
```

[the remark this answers is not in the document]
Comment (gnikit) [reply pages/how-to/compute-pi-openmp.md:12]: Well I just realised I have mistyped the **directory** name not the path :laughing:

**Pull request merged**